python_modules/libraries/dagster-dbt/dagster_dbt/cli/utils.py

```diff
@@ -119,8 +119,10 @@ def execute_cli(
             except json.JSONDecodeError:
                 pass
             else:
-                message = json_line.get("message", json_line.get("msg", message))
-                log_level = json_line.get("levelname", json_line.get("level", "debug"))
+                # in rare cases, the loaded json line may be a string rather than a dictionary
+                if isinstance(message, dict):
```

Comment:
what happens if it _is_ a string? should we parse it out?

Reply:
by default, message is set to the bare string (and log level to "info"). so if it's a string, the raw json line will be emitted (seems ok to me).